Zapper-fi/studio
# feat(vendor-finance): Adds support for version 2 lending pools

## Description
I have implemented support for Vendor protocol v2. While most of the changes are taken from the existing code, there is a notable difference. The updated code utilizes a different subgraph to retrieve the v2 pools, the response of which is similar to the existing v1 response, but with some differences.


## Checklist


## How to test?
1) Run: `pnpm dev`
2) Go to the link: http://localhost:5001/apps/vendor-finance/balances?addresses%5B%5D=0xa09D5BA6f2463ABD0429faB9F4aaA920B185387b&network=arbitrum

You should see `"label":"Lending Pools V2"` somewhere in the response
